### PR TITLE
Meeting notes from JS-IPFS Dev meeting

### PR DESCRIPTION
**Pull request by vmx**
@diasdavid  Please especially check my Waffle Board comments at the bottom if they are correct.

### CODE REVIEW

File: meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+   - The scheme is for the current point in time. So those things will be in "done" and "next" at the same time
+ - How to verify that PRs work if they have other dependencies?
+    - Add the information about the dependencies into the PR and the reviewers will test it locally
+
+ 
+### Other notes
+
+ - @diasdavid proposes having a daemon pool for the tests, so tests run faster
+ - Please use the Waffle Board as intended. See the first meeting notes for more information on that
+   - Introducing a new label for being blocked
```
Comment (diasdavid) [inline meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md]: https://github.com/ipfs/wg-qa-testing-dx/issues/34

File: meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+   - "In progress" meansyou are actively working on an issue and are not blocked
+  - OKRs: @diasdavid will open an PR which will be used to discuss them
```
Comment (diasdavid) [inline meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md]: https://github.com/ipfs/js-ipfs/pull/1281

File: meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  - @vmx
+  - @zcstarr
+  - @jacobheun
+  - @fsdiogo
+  - @hugomrdias
+  - @Mr0grog
+  - @victorbjelkholm
+  - @achingbrain
+  - @diasdavid
+- **Recording:** _add link to recording once it's online_
```
Comment (diasdavid) [inline meeting-notes/2018-03-26-js-ipfs-dev-team-weekly.md]: I've pasted the url on cryptpad and the issue. Can you c&p it here?

**Comment by vmx** [comment]
@diasdavid I updated it with all the links. It's ready to get squashed.

**Pull request merged**